src/test/java/com/puppycrawl/tools/checkstyle/grammars/javadoc/ParseTreeBuilder.java

```diff
@@ -5772,4 +5772,61 @@ public static ParseTree treeCustomTagWithDot()
         return javadocContext0;
     }
 
+    public static ParseTree treeInlineCustomJavadocTag()
```

Comment:
@romani We should remove this way of validation and switch to verifying via `AstTreeStringPrinter` like we do with `verifyAst`. Reading, understanding, and modifying custom written trees is not easy.
I will make an issue for this.